gifted_build/src/server/start_servers.js

```diff
@@ -0,0 +1,60 @@
+const express = require('express');
+const bodyParser = require('body-parser')
+const session = require('express-session')
+const sessionFileStore = require('session-file-store')(session)
+const passport = require('passport')
+const app = express();
+const server = express();
+const path = require('path')
+
+module.exports.startServers = mode => {
+    if (mode === "DEVELOPMENT") {
```

Comment:
This should not be placed in a common express file. 

Development and production builds require different dependencies and packages, and the import commands in a common file will require npm to install all in order to run them. We do not want webpack-dev-middleware/webpack-hot-middleware to be required when in production mode.

I highly recommend elevating these commands back into separate dev and prod files respectively.